.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
This line as well as `.husky/.gitignore` and the `.husky/_` directory are created by Husky when creating a hook as outlined in Husky's documentation (`npx husky add .husky/pre-commit`). This file setup was introduced in the newer version of Husky, which is why we didn't have it previously.